### PR TITLE
feat: Remove bookmark button from certain spots

### PR DESCRIPTION
**Pull request by AmarTrebinjac**
## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://mi-662.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/widgets/SimilarPosts.tsx [hunk 1]
```diff
@@ -52,58 +54,66 @@ const DefaultListItem = ({
...
+          </div>
+        ) : (
+          <PostEngagementCounts
+            upvotes={post.numUpvotes}
+            comments={post.numComments}
+            className="text-text-tertiary"
+          />
+        )}
+      </div>
+      {!isPlus && (
```
Comment (AmarTrebinjac) [inline packages/shared/src/components/widgets/SimilarPosts.tsx:99]: This and adding the "isPlus" variable at the top is the only change. A lot of green due to changing return from () to {}

File: packages/shared/src/components/widgets/SimilarPosts.tsx [hunk 2]
```diff
@@ -52,58 +53,66 @@ const DefaultListItem = ({
...
+          </div>
+        ) : (
+          <PostEngagementCounts
+            upvotes={post.numUpvotes}
+            comments={post.numComments}
+            className="text-text-tertiary"
+          />
+        )}
+      </div>
+      {!showPlusSubscription && (
```
Comment (AmarTrebinjac) [inline packages/shared/src/components/widgets/SimilarPosts.tsx:98]: adding the hook at the top and this is the only real change. A lot of diff due to changing component from implicit to explicit return.

**Review by capJavert — APPROVED** [review]
Looks good, just wondering if this should be merged before other bookmark stuff.

**Comment by AmarTrebinjac** [comment]
> Looks good, just wondering if this should be merged before other bookmark stuff.

Nah, we created a intermediate branch. Just forgot to set it. Good catch!

**Review by rebelchris — COMMENTED** [review]
Why did you need all the tab/feednav changes, not sure I follow those?
We only removing the bookmark button here no?

**Comment by AmarTrebinjac** [comment]
> Why did you need all the tab/feednav changes, not sure I follow those? We only removing the bookmark button here no?

They're just cause I switched target branch from main to the intermediate. Updated the intermediate branch now

File: packages/shared/src/components/widgets/SimilarPosts.tsx [hunk 3]
```diff
@@ -52,58 +53,66 @@ const DefaultListItem = ({
...
-          <div className="ml-2">{post.trending} devs read it last hour</div>
-        </div>
-      ) : (
-        <PostEngagementCounts
-          upvotes={post.numUpvotes}
-          comments={post.numComments}
-          className="text-text-tertiary"
-        />
+}: PostProps): ReactElement => {
+  const { showPlusSubscription } = usePlusSubscription();
```
Comment (sshanzel) [inline packages/shared/src/components/widgets/SimilarPosts.tsx:57]: In light of the discussion, it seems we don't have to check for Plus now 😅

**Comment by ilasw** [comment]
@AmarTrebinjac we should move the target branch into main at this point, there is no reason to put into the intermediate one ✔️

File: packages/shared/src/components/widgets/SimilarPosts.tsx (hunk 3 above)
Comment (AmarTrebinjac) [reply packages/shared/src/components/widgets/SimilarPosts.tsx]: @sshanzel Done, and also removed tests, variables and functions that are no longer used.

**Review by rebelchris — APPROVED** [review]
Nice cleanup :)

**Review by sshanzel — APPROVED** [review]
Always love seeing code get deleted 🤩

**Pull request merged**